src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psm1

```diff
@@ -177,6 +177,54 @@ function Create-AzApiManagementUserAccount {
 
 Export-ModuleMember -Function Create-AzApiManagementUserAccount
 
+<#
+ .Synopsis
+  Create or update users in Azure API Management.
+
+ .Description
+  Create or update users in an existing Azure API Management instance based on a configuration file.
+
+ .Parameter ResourceGroupName
+  The resource group containing the API Management service.
+
+ .Parameter ServiceName
+  The name of the API Management service located in Azure.
+
+ .Parameter ConfigurationFile
+  The file containing the users and their configuration.
+
+ .Parameter StrictlyFollowConfigurationFile
+  Indicates whether the configuration file should strictly be followed, for example remove the user from groups not defined in the configuration file.
+
+ .Parameter ApiVersion
+  [Optional] The version of the api to be used.
+
+ .Parameter SubscriptionId
+  [Optional] The Id of the subscription containing the Azure API Management service. When not provided, it will be retrieved from the current context (Get-AzContext).
```

Comment:
We sometimes use 'Azure API Management service' and other times 'Azure API Management instance', should we streamline this?

Reply:
Yes we definitely should 👍🏻 
Streamline (also for the other scripts) in [`743bea6` (#422)](https://github.com/arcus-azure/arcus.scripting/pull/422/commits/743bea6b2dc8a2f48bc993ac91362f05a8642e2c)